### PR TITLE
LPS-188837 EVP Automated Test| master

### PR DESCRIPTION
**Pull request by joelrocha68**
[LPS-188838](https://liferay.atlassian.net/browse/LPS-188838)
[LPS-188839](https://liferay.atlassian.net/browse/LPS-188839)
[LPS-188840](https://liferay.atlassian.net/browse/LPS-188840)

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:osb-site-initializer-evp

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "EVP Portal");
+		}
+	}
+
+	@description = "This is a test for LPS-164731. Verify that Reports page is not visible on top menu for Employee users."
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase:38]: Hey @joelrocha68 , Could you change the test description as previously align?

Comment (renato-araujo-liferay) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase:38]: Sugestion: "LPS-164731 - Reports bar Should not be visible on top menu for Employee users."

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		task ("When he goes to EVP Site") {
+			EVPUtils.gotoEVP();
+		}
+
+		task ("Then he can not see Reports Page on top menu") {
+			EVPNavigator.navItemNotPresent(navItem = "Reports");
+		}
+	}
+
+	@description = "This is a test for LPS-164731. Verify that user can access Reports page through 'Reports' Tab on top menu and it is visible to EVP Finance"
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase:56]: Hey @joelrocha68 , Could you change the test description as previously align?



Sugestion: "LPS-164731 - Finance user should be able to see the Report card through 'Reports' Tab on top menu"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+			EVPNavigator.goToNavItem(navItem = "Reports");
+		}
+
+		task ("Then he can see the cards present in the page") {
+			EVPUtils.assertCardTitle(listcardTitle = "EVP Requests Listing,EVP Financial Report");
+		}
+	}
+
+	@description = "This is a test for LPS-164731. Verify that user can access Reports page through 'Reports' Tab on top menu and it is visible to EVP Manager"
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase:76]: Hey @joelrocha68 , Could you change the test description as previously align?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+	@priority = 5
+	test ManagerCanAccessReportPage {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase:78]: Please check with gabi if this testcase name rings a bell

**Review by renato-araujo-liferay — CHANGES_REQUESTED** [review]
Joel, Please check these previous comments.

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+		}
+
+		task ("When he goes to Reports Page") {
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Reports");
+		}
+
+		task ("Then he can see the cards present in the page") {
+			EVPUtils.assertCardTitle(listcardTitle = "EVP Requests Listing,EVP Financial Report");
```
Comment (LuanaKSilva) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase:133]: After changing the macro name, remember to change the macro calls.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
```
Comment (LuanaKSilva) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase:153]: Same comment as above.

**Review by LuanaKSilva — CHANGES_REQUESTED** [review]
Hey @joelrocha68, thanks for working on it!
I have requested some adjustments, please take a look.

**Comment by joelrocha68** [comment]
ci:forward

**Comment by joelrocha68** [comment]
ci:close

**Comment by joelrocha68** [comment]
Resent  #4412

**Pull request closed without merging**